wagtailio/areweheadlessyet/models.py

```diff
@@ -0,0 +1,37 @@
+from django.db import models
+from wagtail.admin.edit_handlers import FieldPanel, MultiFieldPanel, StreamFieldPanel
+from wagtail.core.fields import RichTextField, StreamField
+from wagtail.core.models import Page
+
+from wagtailio.areweheadlessyet.blocks import HomePageBlock
+
+
+class AreWeHeadlessYetHomePage(Page):
+    THUMBS_UP = "\U0001f44D"
+    THUMBS_DOWN = "\U0001f44E"
+    ICON_CHOICES = [
+        (THUMBS_UP, THUMBS_UP),
+        (THUMBS_DOWN, THUMBS_DOWN),
+    ]
+    strapline_icon = models.CharField(
+        max_length=255,
```

Comment:
```suggestion
        max_length=1,
```

Although, with unicode codepoints, 1 may not actually be correct. Either way 255 is probably way too high